tests/testutil/service.go

```diff
@@ -202,6 +199,23 @@ func (ts *TestService) GetRunningPods() string {
 	return out
 }
 
+// GetKubeCurrentWeight takes in a revision value and retrieves the actual current weight, not the spec (end-goal) weight
+func (ts *TestService) GetKubeCurrentWeight(revision string) int {
```

Comment:
You are already on the service object so you don't need to pass in revision, use `ts.Version`.

Reply:
I might need to rewrite this whole thing, or create an `app.go` helper file. This really acts on the "apps.rio.cattle.io" resource. 
For example, when called in one of the tests, it's called from the same service object:
```
assert.Equal(t, 95, service.GetKubeCurrentWeight(service.Version))
assert.Equal(t, 5, service.GetKubeCurrentWeight(stagedService.Version))
```
I'm kind of leaning towards a design of something like `service.GetApp()` and then have a `TestApp` object to call something like this on. Thoughts?

Reply:
Shouldn't go app route because most likely app won't exist soon, it will just all be services linked together. 
Why can't you call `stagedService.GetKubeCurrentWeight()` ? service and stagedService are on same app.

Reply:
... because that's too simple and I overthought it 👍